lib/presets.js

```diff
@@ -428,6 +428,63 @@ module.exports = {
           return callback(error);
         }
       });
+    },
+
+    /**
+     * @summary Update quoted version immediately following a regex
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} [options.baseDir] - directory to prefix onto options.file
+     * @param {String} options.file - file to modify
+     * @param {String} options.regex - regex leading up to the quoted version string
+     * @param {String} [options.regexFlags] - any modifier flags to apply to the regex
+     * @param {String} cwd - current working directory
+     * @param {String} version - version
+     * @param {Function} callback - callback (error)
+     * @returns {null}
+     *
+     * @example
+     * presets.updateVersion.quoted({file: "myfile.h", regex: /^VERSION\s+=\s+/}, process.cwd(), '1.0.0', (error) => {
+     *   if (error) {
+     *     throw error;
+     *   }
+     * });
+     */
+    quoted: (options, cwd, version, callback) => {
+      _.defaults(options, {
+        baseDir: '.',
+        regexFlags: ''
```

Comment:
I am not sure this is really needed as `undefined` is the default for `RegExp`. But then having a value can be a better practice for default options.

Reply:
Nope, default is an empty string
```js
$ node
> RegExp().flags
''
```

(and if I change this to `undefined` my new regex-flags-merging code breaks :laughing: )